lib/ansible/module_utils/netapp.py

```diff
@@ -45,6 +49,12 @@
     HAS_NETAPP_LIB = False
 
 import ssl
+import logging
```

Comment:
Please use Ansible's build in logging, see `module_utils/basic.py`

Reply:
Thanks for letting me know about the built in Ansible logging. I went ahead and replaced the python logging with it.